src/hooks/index.js

```diff
@@ -12,6 +12,8 @@ exports.ifNotLocal = function (hookForRemoteRequests) {
 		if (typeof (hook.params.provider) != 'undefined') {	// meaning it's not a local call
```

Comment:
it's probably easier to understand if you use the positive condition and swap then with else

Reply:
True, i solved it.